staging/src/k8s.io/client-go/discovery/cached/memory/memcache.go

```diff
@@ -79,7 +80,7 @@ func isTransientError(err error) bool {
 		return true
 	}
 
-	if t, ok := err.(errorsutil.APIStatus); ok && t.Status().Code >= 500 {
+	if t, ok := err.(errorsutil.APIStatus); ok && t.Status().Code >= http.StatusInternalServerError {
```

Comment:
these`>= constant` cases IMO make the code harder to read. Would revert them. `== constant` is fine.